test/augmentation/test_auto_operation.py

```diff
@@ -109,9 +110,19 @@ def test_smoke(self, policy):
         in_tensor = torch.rand(10, 3, 50, 50, requires_grad=True)
         aug(in_tensor)
 
-    def test_reproduce(self):
+    def test_transform_mat(self, device, dtype):
```

Comment:
@johnnv1 Can you check if float64 issues are fixed or not?

I do not understand how does different input dtypes affect the computation though. But it seems to be working on my machine.

Reply:
The tests are also passing on CI, so looks fine